File: Adapter/Python/SquarePeg.py
import math

class SquarePeg():
  def __init__(self, width:float) -> None:
    self.__width__ = width

  @property
  def Width(self) -> float:
    return self.__width__

  @property
  def Radius(self) -> float:
    return self.Width / math.sqrt(2.0)

File: Adapter/CSharp/SquarePeg.cs
using System;

namespace CSharp
{
  class SquarePeg
  {
    public double Width { get; private set; }
    //public double Radius { get => Width / Math.Sqrt( 2.0 ); }

    public SquarePeg( double width )
    {
      Width = width;
    }
  }
}
